dtfs-central-api/src/types/utilisation-reports.ts

```diff
@@ -1,10 +1,10 @@
 import { ObjectId } from 'mongodb';
-import { ValuesOf } from '@ukef/dtfs2-common';
+import { ValuesOf, UtilisationReportReconciliationStatus } from '@ukef/dtfs2-common';
 import { IsoMonthStamp, MonthAndYear } from './date';
-import { UTILISATION_REPORT_RECONCILIATION_STATUS, UTILISATION_REPORT_HEADERS } from '../constants';
+import { UTILISATION_REPORT_HEADERS } from '../constants';
 import { Currency } from './currency';
 
-export type UtilisationReportReconciliationStatus = ValuesOf<typeof UTILISATION_REPORT_RECONCILIATION_STATUS>;
+export { UtilisationReportReconciliationStatus };
```

Comment:
We can just do that, but I wanted to avoid refactoring all of that at once. This PR will enable us to move types in following PRs to directly import from `@ukef/dtfs2-common`, but if I refactored all the imports of the types I moved to common this PR would become significantly bigger